Source/LightboxViewCell.swift

```diff
@@ -0,0 +1,21 @@
+import UIKit
+
+public class LightboxViewCell: UICollectionViewCell {
+
+  public static let reuseIdentifier: String = "LightboxViewCell"
```

Comment:
@richardoti I feel like it belongs to the cell, so you can avoid code duplication in other view controllers if you want to use this cell again. @zenangst What do you think?